Makd.mak

```diff
@@ -651,11 +651,10 @@ d2conv: $O/d2conv.stamp
 
 $O/d2conv.stamp: $C
 	$Vfind $C -type f -regex '^.+\.d$$' > $@
-ifeq "$(shell d1to2fix --help | grep -- --input)" ""
+	@if d1to2fix --help | grep -- --input; then \
```

Comment:
Yes. I want only the statement matching current d1to1fix version to be visible in verbose mode